CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [v1.2.3] (Mar 4 2024)
+#### Chore:
+- Injected `chat-ai-widget-key` key-value pair into the user agent parameter during build time.
```

Comment:
How about just copying changes of `v1.2.2`?

Reply:
or... improved stability? 🌝

Reply:
Oh yeah I know your intent. Then how about just mentioning it's just an internal change?

Reply:
great~